docs/index.rst

```diff
@@ -11,6 +11,17 @@ data from the event model into a Streamz graph while tracking data
 provenance and workflow execution.
 
 
+Installation
+============
+
+To install SHED use the conda-forge channel for the conda package
```

Comment:
do you just support installation using conda?  If yes, say so.  If no, then give instructions how to install using othe rmethods.

Reply:
Currently this is conda or source this isn't on pypi.